CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 # Changelog
 
 ## Version 0.39.0
-_In development_
+
+<!-- ([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.38.0...v0.39.0)) -->
+
+### Highlights
+
+We've made updates to our documentation, beginning with the completion of the first section of the Pauli Twirling user guide, which offers a comprehensive introduction to this feature.
+Additionally, we've added a new tutorial on CDR (Clifford Data Regression) with [Qrack](https://github.com/unitaryfund/qrack/), providing users with an in-depth look at how to integrate these tools effectively.
```

Comment:
Might be nice to mention that Qrack is a near-Clifford simulator, so the power of (mostly) efficient simulation in training can be realized.

Reply:
Great suggestion, thanks! Changed in [`fc0a282`](https://github.com/unitaryfund/mitiq/pull/2457/commits/fc0a2825d1d2c82d9d0a3f49df8eedc86b462108).